virtcontainers/container.go

```diff
@@ -762,7 +764,13 @@ func (c *Container) create() (err error) {
 	}
 	c.process = *process
 
-	// If this is a sandbox container, store the pid for sandbox
+	cgroupsPath, err := c.newCgroups()
+	if err != nil {
+		return
+	}
+	c.state.CgroupPath = cgroupsPath
```

Comment:
Why don't put this line into `c.newCgroups()` ?

Reply:
sounds good